app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkFragment.kt

```diff
@@ -186,42 +185,34 @@ class BookmarkFragment : Fragment(), BackHandler, AccountObserver {
                 true
             }
             R.id.open_bookmarks_in_new_tabs_multi_select -> {
-                getSelectedBookmarks().forEach { node ->
-                    node.url?.let {
-                        context?.components?.useCases?.tabsUseCases?.addTab?.invoke(it)
-                    }
+                openItemsInNewTab { tabsUseCases, node ->
+                    node.url?.let { tabsUseCases.addTab.invoke(it) }
                 }
 
-                (activity as HomeActivity).browsingModeManager.mode = BrowsingModeManager.Mode.Normal
-                (activity as HomeActivity).supportActionBar?.hide()
                 nav(R.id.bookmarkFragment, BookmarkFragmentDirections.actionBookmarkFragmentToHomeFragment())
                 metrics?.track(Event.OpenedBookmarksInNewTabs)
                 true
             }
+            R.id.open_bookmarks_in_private_tabs_multi_select -> {
```

Comment:
Do we have to change the mode to private here as well? It looks like openItemsInNewTab will force it to normal mode?

Reply:
Oops 😅